OpenOversight/tests/test_functional.py

```diff
@@ -75,23 +84,21 @@ def test_user_can_get_to_complaint(mockdata, browser):
     assert "File a Complaint" in title_text
 
 
-def test_officer_browse_pagination(mockdata, browser):
+def test_officer_browse_pagination(mockdata, browser, engine):
     dept_id = 1
-    total = Officer.query.filter_by(department_id=dept_id).count()
-    perpage = BaseConfig.OFFICERS_PER_PAGE
+    total = list(engine.execute('select count(1) from officers where department_id = 1'))[0][0]
+    perpage = DevelopmentConfig.OFFICERS_PER_PAGE
```

Comment:
honestly couldn't figure out how to get `Officer.query` to use my new `engine` pointing to the docker container... if anyone has an idea, lemme know.